readme.md

```diff
@@ -27,7 +27,23 @@ console.log(wrapAnsi(input, 20));
 
 ## API
 
-### wrapAnsi(input, columns)
+### wrapAnsi(input, columns, opts)
+
+wrap words to the specified column width. By default the wrap is
+`soft`, so long words may extend past the column length.
+
+the following options can be provided:
+
+* `hard`: `boolean`, should hard wrapping be enabled?
+
+### wrapAnsi.hard(input, columns)
+
+long words will be broken up so that they do not extend past the column width.
+Equivalent to `{hard: true}`.
+
+### wrapAnsi.soft(input, columns)
```

Comment:
I can see how using separate methods could get unwieldy if other options are needed in the future, unless you use a chaining approach like chalk. Probably safer to just use options. :(